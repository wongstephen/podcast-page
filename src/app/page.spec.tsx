import { render } from '@testing-library/react';
import Page from './page';

jest.mock('next/navigation', () => ({
  usePathname: jest.fn(() => '/podcast'),
}));

describe('sanity', () => {
  it('renders', () => {
    const { baseElement } = render(<Page />);
    expect(baseElement).toBeTruthy();
  });

  it('matches snapshot', () => {
    const { container } = render(<Page />);
    expect(container).toMatchSnapshot();
  });
});
