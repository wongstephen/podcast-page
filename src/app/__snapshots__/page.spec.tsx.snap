// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`sanity matches snapshot 1`] = `
<div>
  <header
    class="container"
  >
    <div
      class="wrapper"
    >
      <div
        class="container"
      >
        <p
          class="phoneText"
        >
          +1 888 240 6923
        </p>
        <a
          class="contactLink"
          href="#"
        >
          Contact Us
        </a>
        <div
          class="buttonsContainer"
        >
          <button
            class="container"
            data-appearance="outline"
            data-size="small"
          >
            Login
          </button>
          <button
            class="container"
            data-appearance="outline-secondary"
            data-size="small"
          >
            FR
          </button>
        </div>
      </div>
    </div>
    <div
      class="logoContainer"
    >
      <svg
        class="logo"
        fill="none"
        viewBox="0 0 292 82"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          clip-rule="evenodd"
          d="M41.813 0c8.7 0 16.397.696 21.08 1.764 9.335 1.935 16.687 9.195 18.646 18.416 1.08 4.625 1.786 12.226 1.786 20.82s-.705 16.196-1.786 20.82c-1.959 9.22-9.31 16.482-18.646 18.416C58.21 81.304 50.513 82 41.813 82c-8.702 0-16.399-.696-21.08-1.764C11.395 78.302 4.044 71.04 2.085 61.82 1.006 57.196.3 49.594.3 41s.705-16.195 1.786-20.82c1.959-9.22 9.31-16.482 18.646-18.416C25.415.696 33.112 0 41.812 0Zm5.51 59.484a6.279 6.279 0 0 1 4.497 3.68c.26.614.892.99 1.563.93l.512-.049c3.253-.321 5.81-.769 7.297-1.292a2.613 2.613 0 0 0 1.708-2.67l-.003-.03a1.224 1.224 0 0 1-.005-.085c-.254-3.358-2.313-11.22-5.388-20.193-3.516-10.258-7.192-18.563-8.85-20.183a3.559 3.559 0 0 0-1.404-.89l-.018-.007c-1.088-.364-3.095-.609-5.396-.609-2.297 0-4.304.245-5.394.61l-.015.005a3.558 3.558 0 0 0-1.404.89c-1.658 1.62-5.335 9.925-8.85 20.184-3.076 8.973-5.135 16.835-5.389 20.193 0 .027-.002.055-.005.085l-.003.031-.008.138-.001.083a2.613 2.613 0 0 0 1.717 2.448c1.488.523 4.048.972 7.306 1.293l.448.042a1.62 1.62 0 0 0 1.636-.963 6.28 6.28 0 0 1 4.48-3.641c1.218-.279 3.22-.46 5.484-.46s4.266.181 5.484.46Z"
          fill="#00BD82"
          fill-rule="evenodd"
        />
        <path
          clip-rule="evenodd"
          d="M132.995 26.727h7.939v35.685h-7.939v-6.715c-2.48 4.277-6.733 6.73-12.262 6.73-9.215 0-16.587-7.992-16.587-17.877 0-9.885 7.372-17.807 16.587-17.807 5.529 0 9.782 2.454 12.262 6.73v-6.745Zm118.857 0h7.939v35.685h-7.939v-6.715c-2.481 4.277-6.734 6.73-12.263 6.73-9.215 0-16.586-7.992-16.586-17.877 0-9.885 7.371-17.807 16.586-17.807 5.529 0 9.782 2.454 12.263 6.73v-6.745ZM122.611 54.97c6.025 0 10.562-4.486 10.562-10.446 0-5.889-4.537-10.376-10.562-10.376-5.883 0-10.561 4.628-10.561 10.376 0 5.82 4.678 10.446 10.561 10.446Zm118.441 0c6.025 0 10.561-4.486 10.561-10.446 0-5.889-4.536-10.376-10.561-10.376-5.884 0-10.562 4.628-10.562 10.376 0 5.82 4.678 10.446 10.562 10.446Zm-88.231-44.684c2.764 0 5.103 2.313 5.103 4.977 0 2.805-2.339 5.048-5.103 5.048a5.04 5.04 0 0 1-5.033-5.048c0-2.664 2.268-4.977 5.033-4.977Zm-3.899 16.347h7.868v35.685h-7.868V26.633Zm32.583.015c1.516 0 2.975.223 4.027.619.619.232 1.166.49 1.59.707l-3.202 6.63a11.212 11.212 0 0 0-3.195-.454c-3.828 0-8.08 3.155-8.08 9.184v18.984h-7.868V26.633h7.868v5.624c1.559-3.295 5.457-5.609 8.86-5.609Zm39.515 26.29c-2.622 5.68-8.718 9.395-15.948 9.395-10.278 0-18.146-7.642-18.146-17.807 0-10.166 7.797-17.878 18.075-17.878 6.734 0 12.404 3.295 14.956 8.343l-6.663 3.716c-1.418-2.734-4.536-4.557-8.293-4.557-5.954 0-10.278 4.487-10.278 10.376 0 5.819 4.395 10.305 10.349 10.305 4.04 0 7.371-1.892 8.931-5.047l7.017 3.155Zm54.625-42.652v52.032h-7.868V10.286h7.868Zm15.855 0v52.032h-7.868V10.286h7.868Z"
          fill="#012635"
          fill-rule="evenodd"
        />
      </svg>
      <nav>
        <div
          class="mobileMenuContainer"
        >
          <button
            aria-expanded="false"
            aria-label="Toggle navigation menu"
            class="container menuButton"
            data-appearance="transparent"
            data-size="none"
          >
            <p
              class="mobileMenuText"
            >
              Menu
            </p>
            <svg
              aria-hidden="true"
              class="menuIcon"
              fill="none"
              height="9"
              width="14"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M0 1h14M0 8h14"
                stroke="currentColor"
                stroke-width="2"
              />
            </svg>
            <span
              class="sr-only"
            >
              Open menu
               
            </span>
          </button>
        </div>
      </nav>
    </div>
  </header>
</div>
`;
